pkg/kademlia/config.go

```diff
@@ -41,6 +43,40 @@ type OperatorConfig struct {
 	Wallet string `user:"true" help:"operator wallet adress" default:""`
 }
 
+// Verify verifies whether operator config is valid.
+func (c OperatorConfig) Verify(log *zap.Logger) error {
+	if err := isOperatorEmailValid(log, c.Email); err != nil {
+		return err
+	}
+
+	if err := isOperatorWalletValid(log, c.Wallet); err != nil {
+		return err
+	}
+
+	log.Sugar().Info("Operator email: ", c.Email)
```

Comment:
Can we show this only if the email is actually set? Currently I see this:
```
2019-01-23T11:45:17.682+0200	WARN	Operator email address isn't specified.
2019-01-23T11:45:17.682+0200	INFO	Operator email: 
```
Perhaps, move the logging into the `isOperatorEmailValid` function.

Reply:
Although the config `Verify` part is one of the thing I'm unsure about... there might be a generally better way to handle this.